site/gulpfile.js

```diff
@@ -14,6 +14,11 @@ const responsive    = require('gulp-responsive');
 
 const {swagList, swagImages} = require('./get-data');
 
+let manifest = {
```

Comment:
Can we move this after the `RESIZE_OPTS` const? I like to follow the design pattern where you define constants first